backend/src/main/java/com/mcmasterbaja/analyzer/BullshitAnalyzer.java

```diff
@@ -1,10 +1,14 @@
 package com.mcmasterbaja.analyzer;
 
+import com.mcmasterbaja.annotations.OnAnalyzerException;
+import com.mcmasterbaja.annotations.OnStorageException;
+import com.mcmasterbaja.exceptions.InvalidHeaderException;
 import com.opencsv.CSVReader;
 import com.opencsv.ICSVWriter;
-import com.opencsv.exceptions.CsvValidationException;
-import java.io.IOException;
+import lombok.SneakyThrows;
 
+@OnAnalyzerException
+@OnStorageException
```

Comment:
Why is OnStorageException here?

Reply:
Also with recent discussion probably no need to have any OnStorageException anymore, since all errors in there are too specific to generalize